cmd/kfcluster/kfcluster_create.go

```diff
@@ -73,7 +85,6 @@ var kfcCreateCmd = &cobra.Command{
 			NetworkID:  network.ID,
 			Size:       size,
 			FirewallID: firewallID,
-			Region:     client.Region,
```

Comment:
why are we removing it?

Reply:
We pass it before sending a create request - https://github.com/civo/civogo/blob/master/kfcluster.go#L113
Same way we do for k3s clusters. No need to add it here.